src/types/services/nnid/regions.ts

```diff
@@ -0,0 +1,38 @@
+export interface Regions {
+	id: number;
+	iso_code: string;
+	name: string;
+	translations: Translations;
+	regions?: (RegionsEntity)[] | null;
+}
+
+export interface Translations {
+	japanese: string;
+	english: string;
+	french: string;
+	german: string;
+	italian: string;
+	spanish: string;
+	chinese_simple: string;
+	korean: string;
+	dutch: string;
+	portuguese: string;
+	russian: string;
+	chinese_traditional: string;
+	unknown1: string;
+	unknown2: string;
+	unknown3: string;
+	unknown4: string;
+}
+
+export interface RegionsEntity {
+	id: number;
+	name: string;
+	translations: Translations;
+	coordinates: Coordinates;
+}
+
+export interface Coordinates {
+	latitude: number;
+	longitude: number;
+}
```

Comment:
The `Coordinates` type doesn't seem to be used outside of the `RegionsEntity` type, so it can just be merged into it

```suggestion
	coordinates: {
		latitude: number;
		longitude: number;
	};
}
```